SECURITY.md

```diff
@@ -0,0 +1,59 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+To report a vulnerability, either:
+
+1. Report it on Github directly you can follow the procedure described
+[here](https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing/privately-reporting-a-security-vulnerability)
+and:
+    - Navigate to the security tab on the repository
+    - Click on 'Advisories'
+    - Click on 'Report a vulnerability'
+
+2. Send an email to `security@crossplane.io` detailing the issue and steps
```

Comment:
@jbw976 suggestions on how to get this email address in place?

Reply:
I can follow up to make this email address happen after merging this PR - just remind me :)